static/app/components/dropdownMenu.tsx

```diff
@@ -46,9 +45,7 @@ type Props = {
   minWidth?: number;
   onClose?: () => void;
   size?: MenuItemProps['size'];
-} & AriaMenuOptions<MenuItemProps> &
-  Partial<OverlayProps> &
-  Partial<AriaPositionProps>;
```

Comment:
This is where the unused props come from. `OverlayProps` and `AriaPositionProps` are remnants from when we still used `react-aria`'s `useOverlayPosition`.